server/src/Metrics.hs

```diff
@@ -51,9 +55,19 @@ createAndRegisterIcepeakMetrics = IcepeakMetrics
                               "Total number of bytes written to the journal so far."))
   <*> register (gauge
     (Info "icepeak_subscriber_count" "Number of websocket subscriber connections."))
+  <*> register (counter (Info "icepeak_internal_queue_items_added"
+                              "Total number of items added to the queue."))
+  <*> register (counter (Info "icepeak_internal_queue_items_removed"
+                              "Total number of items removed from the queue.."))
+  <*> register (histogram (Info "icepeak_sync_duration" "Duration of a Sync command.")
+                          syncBuckets)
+  <*> register (histogram (Info "icepeak_http_req_handling_duration"
+                                "Duration of the handling of an HTTP request.")
+                          defaultBuckets)
```

Comment:
It might be a nice idea to split those per method and status, like the requestCounter metric in here.

Since a histogram also counts, I think we can actually deprecate the request counter metric eventually.